src/user/serializers.py

```diff
@@ -1054,11 +1054,48 @@ class DynamicAuthorProfileSerializer(DynamicModelFieldSerializer):
     works_count = SerializerMethodField()
     citation_count = SerializerMethodField()
     summary_stats = SerializerMethodField()
+    open_access_pct = SerializerMethodField()
+    achievements = SerializerMethodField()
+    headline = SerializerMethodField()
 
     class Meta:
         model = Author
         fields = "__all__"
 
+    def get_headline(self, author):
```

Comment:
This method will be rewritten over the next few weeks. This is initial version that takes into account authors with no headline